syntax = "proto3";
package packet_cs;
import "base_type.proto";

message MsgReqLoginCL
{
    string account_name = 1;
    string password = 2;
}

message MsgLoginResultLC
{
    int32 result = 1;
    string token = 2;
    repeated pb_base.ServerProfile servers = 3;
}

message PktGameLoginReq
{
    string account_name = 1;
    string session_key = 2;
    string ip_address = 3;
}

message PktRoleListAck
{
    repeated string role_name_list = 1;
}